BundesIdent/Managers/ChangePINDebugSequence.swift

```diff
@@ -4,84 +4,84 @@ import Combine
     case runNFCError
-    case runCardSuspended
+    case askForCAN
     case runCardDeactivated
     case runCardBlocked
-    case runCANError
     
     var id: String {
         switch self {
-        case .cancelPINScan: return "cancel"
-        case .cancelCANScan: return "cancel"
+        case .cancel: return "cancel"
         case .changePINSuccessfully: return "changePINSuccessfully"
-        case .runPINError(let remainingAttempts, _): return "runPINError (\(remainingAttempts))"
+        case .runPINError(let remainingAttempts): return "runPINError (\(remainingAttempts))"
         case .runNFCError: return "runNFCError"
-        case .runCardSuspended: return "runCardSuspended"
+        case .askForCAN: return "askForCAN"
         case .runCardDeactivated: return "runCardDeactivated"
         case .runCardBlocked: return "runCardBlocked"
-        case .runCANError: return "runCANError"
         }
     }
     
     static func defaultActions(card: Card) -> [ChangePINDebugSequence] {
         var actions: [ChangePINDebugSequence] = [
-            .cancelPINScan,
+            .cancel,
             .changePINSuccessfully,
-            .runCardSuspended,
+            .askForCAN,
             .runNFCError,
             .runCardDeactivated,
             .runCardBlocked
         ]
         
         if card.remainingAttempts >= 0 {
-            actions.append(.runPINError(remainingAttempts: card.remainingAttempts, cancelAction: .pin))
+            actions.append(.runPINError(remainingAttempts: card.remainingAttempts))
         }
         
         return actions
     }
     
     func run(card: inout Card, subject: PassthroughSubject<EIDInteractionEvent, IDCardInteractionError>) -> [ChangePINDebugSequence] {
         switch self {
-        case .cancelPINScan:
+        case .cancel:
             subject.send(.cardInteractionComplete)
-            subject.send(.requestChangedPIN(remainingAttempts: nil, pinCallback: { _, _ in
-                subject.send(.requestCardInsertion({ _ in }))
-            }))
-            return ChangePINDebugSequence.defaultActions(card: card)
-        case .cancelCANScan:
-            subject.send(.cardInteractionComplete)
-            subject.send(.requestPINAndCAN({ _, _ in
-                subject.send(.requestCardInsertion({ _ in }))
-            }))
-            return [
-                .cancelCANScan,
-                .changePINSuccessfully,
-                .runCANError,
-                .runNFCError,
-                .runCardDeactivated,
-                .runCardBlocked
-            ]
+            
+            if card.remainingAttempts >= 2 {
+                subject.send(.requestChangedPIN(remainingAttempts: nil, pinCallback: { _, _ in
+                    subject.send(.requestCardInsertion({ _ in }))
+                }))
+                return ChangePINDebugSequence.defaultActions(card: card)
+            } else if card.remainingAttempts == 1 {
+                subject.send(.cardInteractionComplete)
```

Comment:
Second `subject.send(.cardInteractionComplete)` in a row?

Might be more elegant to convert this `.cancel` case with `if` inside into:
``` swift
case .cancel where card.remainingAttempts >= 2:
...
case .cancel where card.remainingAttempts == 1:
...
case .cancel:
```
We would have duplication of `subject.send(.cardInteractionComplete)` as before, but it reads cleaner (opinion, just ignore if disagree).